src/components/HashButton.tsx

```diff
@@ -157,8 +157,8 @@ function AccountHashButtonInner({
           textDecoration: "none",
         }}
       >
-        <Tooltip title={hash} enterDelay={500} enterNextDelay={500}>
-          <span>{name ?? truncateHash}</span>
+        <Tooltip title={name ?? hash} enterDelay={500} enterNextDelay={500}>
+          <span>{name ? truncate(name, 9, 9, "...") : truncateHash}</span>
```

Comment:
Let's use the same ligature in `truncate` instead of three periods `...`. 

https://github.com/aptos-labs/explorer/blob/7aea4288545af9cef3599bf8fa1a97ce7b7ea52e/src/pages/utils.tsx#L105